README.md

```diff
@@ -28,5 +28,13 @@ $ cd tictactoe-java
 
 ### Run the game
 
+* Open the project in an IDE (e.g. IntelliJ IDEA)
+* Navigate to `src/main/java/apprenticeship/main.java`
+* Run `main.main()`
+
 
 ### Run the tests
+
+* Open the project in an IDE (e.g. IntelliJ IDEA)
+* Navigate to `src/main/java/apprenticeship/`
+* Run the test suite
```

Comment:
lol. I know, it's not very descriptive. But because I run them on IntelliJ now.  I am just clicking buttons. I don't have a specific command.